src/views/EventPage.vue

```diff
@@ -1,41 +1,36 @@
 <template>
-<div class="page-container">
-  <div class="nav-btns">
-    <img src="../assets/icons/arrow-left.svg" alt="">
-    <img src="../assets/icons/share.svg" alt="">
+  <div class="page-container">
+    <div class="nav-btns">
+      <img src="../assets/icons/arrow-left-light.svg" alt />
+      <img src="../assets/icons/share-light.svg" alt />
```

Comment:
Share button has some weird borders, maybe from cropping?
<img width="117" alt="Screen Shot 2020-02-25 at 3 21 09 AM" src="https://user-images.githubusercontent.com/44561376/75242981-f4ee2e00-577d-11ea-977e-ce64885eed74.png">

Reply:
i'll look into this but i think this only shows up on desktop right?

Reply:
I think its replicable everywhere, this doesnt look like a screen size problem

Reply:
removed share for now cuz it doesn't make sense to put share here